socialhome/streams/app/stores/streamStore.state.js

```diff
@@ -2,21 +2,37 @@ import _get from "lodash/get"
 
 
 export default function () {
+    const content = _get(window, ["context", "content"], undefined)
     const contentIds = []
     const unfetchedContentIds = []
     const contents = {}
+    if (content) {
+        content.replyIds = []
+        content.shareIds = []
+        contentIds.push(content.id)
+        contents[content.id] = content
+    }
     const replies = {}
     const shares = {}
     const streamName = _get(window, ["context", "streamName"], "")
+    const streamSplits = streamName.split("__")
+    const stream = {
+        id: streamSplits.length ? streamSplits[1] : "",
+        isProfile: streamName.startsWith("profile_"),
+        name: streamSplits[0],
+        single: streamSplits[0] === "content",
+    }
 
     return {
+        content,
         contents,
         contentIds,
         hasNewContent: false,
         newContentLengh: 0,
         replies,
         shares,
-        showAuthorBar: streamName.length > 0 ? !streamName.startsWith("profile_") : false,
+        showAuthorBar: !stream.isProfile,
+        stream,
         streamName,
```

Comment:
I think that can be removed, then.

Reply:
The stream name is still used in full also by at least the websocket url. Would be good to refactor things a bit, but leaving this in the state for now since refactoring that isn't really related to this pr.